docs/plugins/syslog.md

```diff
@@ -1,12 +1,12 @@
-## `syslog` plugin
+# `syslog` plugin
 
 The `syslog` plugin receives [syslog](https://en.wikipedia.org/wiki/Syslog) from network devices and outputs a parsed entry.
 
-### Configuration Fields
+## Configuration Fields
 
 | Field              | Default          | Description |                                                                                                                                                                                                  
 | ---                | ---              | ---         |                                                                                                                                                                                                     
-| `listen_port`      | `514`            | Network port to listen on                                                              |                                                     
+| `listen_port`      | `514`            | Network port to listen on. Default ports are 514 for UDP, 601 for TCP, and 6514 for TCP with TLS.|                                                     
```

Comment:
The word default here should be changed to "standard". I was using bad wording with you the other day in Slack.

Might even be best to say something like this, "Standard ports, as defined by IANA, for syslog are: 514/UDP, 601/TCP, and 6514/TCP+TLS"

Reply:
updated description with your suggestion